answers/Maker.md

```diff
@@ -0,0 +1,179 @@
+- **Emergency oracles:** Emergency oracles are selected by MKR holders and have the ability to unilaterally trigger an emergency shutdown. They serve as a last line of defense, should a malicious proposal or governance attack threaten the system.
+
+## Coin distribution
+
+The MKR token was launched on Dec 27, 2017. 1,000,000 MKR were premined. Maker did not ICO. In the early days, tokens were sold strategically by the Maker Foundation to members of the community, with preference given to early contributors to the project. Sales were largely negotiated on an individual basis.
+
+In 2017, the Maker foundation made its first institutional sale to Polychain Capital, a deal which was publicly [negotiated](https://www.reddit.com/r/MakerDAO/comments/5mxwxw/polychain_capital_proposal_to_makerdao/) with community input on the MakerDAO subreddit. Subsequent sales to other institutional investors such as Andreesen Horowitz, Placeholder VC, and others) were modeled on this deal.
+
+Because MKR must be used to pay stability fees, and this MKR is burned upon payment, the supply of MKR is continually decreasing as CDPs are paid off. On Jan 29th, 2019, Rune Christensen estimated on the Unchained podcast that "less than 0.1% of the total supply" had been burned.
+
+
+### Further reading 
+
+[Overview of MakerDAO | Dai](https://github.com/makerdao/awesome-makerdao/blob/master/README.md#governance): a collection of articles, FAQs, and other resources on the official MakerDAO [GitHub](https://github.com/makerdao).
+
+## Governance-focused articles
+
+[Deep dive into MakerDAO Governance](https://medium.com/@banteg/deep-dive-into-makerdao-governance-437c89493203)
+
+[MakerDAO Governance Risk Framework (Part 1)](https://medium.com/makerdao/makerdao-governance-risk-framework-38625f514101)
+
+[MakerDAO Governance Risk Framework (Part 2)](https://medium.com/makerdao/makerdao-governance-risk-framework-fc8939f3d4e9)
+
+[MakerDAO Governance Risk Framework (Part 3)](https://medium.com/makerdao/makerdao-governance-risk-framework-part-3-7a4c620f4077)
+
+[Evaluating MKR Buyback and Burn model](https://medium.com/@TrustlessState/evaluating-mkr-def6d36092bd): Medium Post by David Hoffman
+
+
+# Funding
+
+The Maker Foundation, which employs a team of developers, economists, designers, community managers and others, is funded by sales of the MKR token to investors and stability fees from CDPs when loans are repaid.  MakerDAO and Venture Capital firm L4 have also recently announced the [Stable Fund](https://medium.com/@stablefund/introducing-the-stable-fund-45-million-for-the-makerdao-ecosystem-def79afc4b2a), which will provide $45 million in grants and investments for projects using DAI or the underlying Maker infrastructure.
+
+
+## For the reference node implementation
```

Comment:
You can cut all of this out as it's not relevant to a token on another blockchain.

Reply:
Assuming you mean all the stuff from `For the reference node...` and below (hard to tell because GitHub is highlighting para above it as well). Have just put N/A for these Qs, but can just delete if we want.